Trumpf.Coparoo.Web/Controls/Table/TableRow.cs

```diff
@@ -17,18 +18,15 @@ public class TableRow : ControlObject
         /// <summary>
         /// Gets the enumeration of cell control objects.
         /// </summary>
-        public IEnumerable<TableDataCell> Cells => FindAll<TableDataCell>();
+        public IEnumerable<TableCell> Cells => FindAll<TableCell>();
 
         /// <summary>
         /// Returns the cell element at a specified index in the cells sequence.
         /// </summary>
-        /// <exception cref="System.ArgumentNullException">Cells seqeunce is null.</exception>
+        /// <exception cref="System.ArgumentNullException">Cells sequence is null.</exception>
         /// <exception cref="System.ArgumentOutOfRangeException">Index is less than 0 or greater than or equal to the number of cell elements in the cells sequence.</exception>
         /// <param name="index">The zero-based index of the cell to retrieve.</param>
         /// <returns>The cell element at the specified position in the cells sequence.</returns>
-        public TableDataCell CellAt(int index)
-        {
-            return Cells.ElementAt(index);
-        }
+        public TableCell CellAt(int index) => Cells.ElementAt(index);
```

Comment:
In addition, I would also add a method `CellAt<T>(int index) => Cells.ElementAt(index).As<T>();`.